Demo/Color+Helpers.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+public extension UIColor {
```

Comment:
I don't know any of that :) What is the `copy files` step? And what is `Cartfile.private`